### ISSUE TITLE
Support Anki import

### ISSUE THREAD

**Issue by 12people** [david-swift/Flashcards#5]
### Describe the solution you'd like

It'd be great if Flashcards allowed importing Anki files.

### Describe alternatives you've considered

I suppose Anki could gain support for export to Flashcards instead, but Anki is the more common format here.

### Additional context

Anki is probably the most commonly-used free and open-source flashcard app. As such, there's a lot of Anki decks being shared around and it'd be great to be able to use them inside Flashcards.

**Comment by david-swift** [comment]
Thanks for opening this issue! There is basic support for importing Anki decks since the commit afe3a8bb87edfdfa0f2dcac6648f9c66d3a78cf3, discussed in #1 [here](https://github.com/david-swift/Flashcards/issues/1#issuecomment-1945485480). This works for simple decks, but the difficulty is that Anki supports HTML while Flashcards doesn't. The idea is that Markdown support could be implemented into Flashcards, and the HTML of a deck could then be converted into Markdown during the import process. What do you think about that, @12people?

**Comment by 12people** [comment]
That sounds great!

Only thing to keep in mind here: there should be a message before or after the import about how some aspects of the Anki deck might not be imported correctly (as HTML -> Markdown is lossy), so that users aren't surprised that some things don't look quite right.

**Comment by rugk** [comment]
I agree Anki support in general would be great! As well… this is your main "competitioner" here, which has an established desktop and more importantly mobile app. Of course though FLOSS apps should think more in terms of cooperation, so making this basically an app which works with Anki on desktop would be great, so one can continue to use the mobile Anki apps to learn there/wherever/which whatever device one wants.

Also maybe it makes sense to group all related issues in a GitHub project or at least label or so. Because this in general also applies to:
* https://github.com/david-swift/Memorize/issues/50 -> web support would possibly mean a full synchronization, which would be great!
* https://github.com/david-swift/Memorize/issues/58 -> another Anki import (or is it even a duplicate of this issue?)
